packages/bpk-component-textarea/src/BpkTextarea.tsx

```diff
@@ -16,37 +16,34 @@
  * limitations under the License.
  */
 
-/* @flow strict */
-
-import PropTypes from 'prop-types';
+import type { FC, TextareaHTMLAttributes } from 'react';
 
 import { cssModules } from '../../bpk-react-utils';
 
 import STYLES from './BpkTextarea.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-type Props = {
-  id: string,
-  name: string,
-  value: string,
-  className: ?string,
-  valid: ?boolean,
-  large: ?boolean,
-};
-
-const BpkTextarea = ({
-  className = null,
+export interface BpkTextareaProps extends TextareaHTMLAttributes<HTMLTextAreaElement> {
```

Comment:
seems no need export `BpkTextareaProps` or maybe it can also export to `packages/bpk-component-textarea/index.ts`